### ISSUE TITLE
Secure technical / administrative endpints

### ISSUE THREAD

**Issue by BernhardBln** [factcast/factcast#1654]
From #1162: Also the authorization configuration should be extended in order to not only define who can read/write to what namespace, but also to have some means of access management to more technical/administrative endpoints.

**Comment by BernhardBln** [comment]
This is indeed about grpc endpoints, hence reopening again.

**Comment by uweschaefer** [comment]
Administrative endpoints are:
```
  rpc handshake (MSG_Empty) returns (MSG_ServerConfig) {}
  rpc enumerateNamespaces (MSG_Empty) returns (MSG_StringSet) {}
  rpc enumerateTypes (MSG_String) returns (MSG_StringSet) {}
  rpc currentTime(MSG_Empty) returns (MSG_CurrentDatabaseTime) {}
```

**Comment by uweschaefer** [comment]
closed by #1162
